03b_model/src/run_model.py

```diff
@@ -630,20 +634,21 @@ def run_replicates(n_reps, prepped_model_io_data_file):
         
         run_id = os.path.join(config['run_id'], str(i).rjust(2,'0'))
         
+        seed = i
```

Comment:
I'm not sure I understand what's going on with setting the see in the config file and then resetting it based on the iteration number here. Is this a different seed? and if so, maybe it should get a different name. I'm not sure what happens when you have a global variable name 'seed' and you create a local variable with the same name inside a function. Either way, it would be good to give them different names for clarity.

Reply:
I added some explanatory text in the `model_config.yaml` file, basically, the idea is that the `seed_set` parameter in the config file only pertains to a single model run using the `main()` function. For the `run_replicates()` and `test_hyperparamters()` functions, the seed is set within those functions as the iterator. This way the seed is unique between replicates, but reproducible.

Reply:
Do you think the seed used in the iterations could be named something like `replicate_seed`? The global variable `seed` is visible inside and outside of functions. Then the local variable `seed` is only visible inside the function it is created within. So inside that function, there is some ambiguity about if you are using the global `seed` or the local `seed`. I imagine it is using the local variable, but giving these variables different names would help clear up any ambiguity.

Reply:
Good idea! Changes made